include/evmc/evmc.h

```diff
@@ -865,25 +865,19 @@ enum evmc_revision
      */
     EVMC_ISTANBUL = 7,
 
-    /**
-     * The Berlin revision.
-     *
-     * https://github.com/ethereum/eth1.0-specs/blob/master/network-upgrades/mainnet-upgrades/berlin.md
-     */
+    /// The Berlin revision.
```

Comment:
The other revisions use the `/*` comment style.

Reply:
I'm trying to convert to C++ comment style.